code/style.css

```diff
@@ -1,5 +1,129 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
 
 body {
-  background-color: lightblue;
+ margin: 0;
+ font-family: 'Montserrat', sans-serif;
 }
+
+header {
+  margin: 0px;
+  padding-top: 5px;
+  background-image: url("images/Fraumitstoff.jpg");
+  background-size: cover;
+  height: 1220px;
+  }
+
+h1 {
+  
+  margin-top: 40px;
+  text-align: center;
+  align-items: center;
+  font-size:90px;
+}
+
+.styled-button {
```

Comment:
A minor comment here is to delete the empty lines here to make the indentation consistent, for example on line 27 and 29.